packages/builder/test/fixture-projects/scripts-dir/scripts/2.js

```diff
@@ -1,6 +1,6 @@
-setTimeout(() => {
-  if (runtimeArgs.network !== "custom") {
-    const fs = require('fs')
-    fs.appendFileSync("output.txt", "scripts directory: script 2 executed\n");
-  }
-}, 100);
+const fs = require('fs')
+
+export default async function (runtimeEnv, deployer, accounts) {
+  await new Promise(resolve => setTimeout(resolve, 100));
+  fs.appendFileSync("output.txt", "scripts directory: script 2 executed\n");
+}
```

Comment:
as above, do we need to use `fs`?

Reply:
I check the output of the script this way. 
There is no easy way to capture stdout.

Reply:
It's good to have one tests with `fs`. 
For other tests - isn't it easier to check the return value?

Reply:
When `run.ts` executes multiple scripts there are no return values.
Scripts either pass or fail -- all or some of them.
So there is no way to ensure if by requesting to run scripts 1,2,3 it ran 1st or only the 3rd one.

Reply:
id you run them in the same process, then they can return a value (eg `7`)

Reply:
But this reduces ability to reason about what they do.